client/src/containers/CategoryForm.js

```diff
@@ -1,40 +1,42 @@
-import React, { Component } from 'react';
-import { connect } from 'react-redux';
+import React, { Component } from "react";
+import { connect } from "react-redux";
 
 class CategoryForm extends Component {
   renderList() {
-    return this.props.categories.map((category) => {
-      return <li key={category.name}>
-      <div className="fl w-50 w-50-ns pa2 ">
-        <button className="hover-bg-orange bg-white pv4 h4 w4 br4">
-          <label className="f4 black">{category.name}</label>
-          <br />
-          <img
-            alt=""
-            className="h3 w3 tc"
-            src={category.icon}
-          />
-        </button>
-      </div>
-    </li>
-    })
+    return this.props.categories.map(category => {
+      return (
+        <li key={category.name}>
+          <div className="fl w-50 w-50-ns pa2 ">
+            <button className="hover-bg-orange bg-white pv4 h4 w4 br4">
+              <label className="f4 black">{category.name}</label>
+              <br />
+              <img
+                alt={category.alt}
+                className="h3 w3 tc"
+                src={category.icon}
+              />
+            </button>
+          </div>
+        </li>
+      );
+    });
   }
 
   render() {
     return (
       <div className="mw7 center ph3-ns">
         <div className="cf ph2-ns">
-      <ul>{this.renderList()}</ul>
-    </div>
-    </div>
+          <ul>{this.renderList()}</ul>
+        </div>
+      </div>
     );
   }
 }
 
-function mapStateToProps(state) {
+const mapStateToProps = state => {
```

Comment:
did you read my comment about the return statement? https://github.com/FACN3/easyRep/pull/21#discussion_r167428544
you just do this

```javascript
const mapStateToProps = state => ({
   categories: state.categories
})

Reply:
@Karyum yeah we did... I though this was what you meant... can we leave it like this for the sake of clarity and code readability? I just think that at the stage we are at (trying to understand redux and all) it would be more understandable for us when we read the code to leave it like this... we can always refactor a bit more later on in the project once all the setup is done.

Reply:
@Karyum nevermind ignore my last comment, I think we just understood what you meant :)

Reply:
nope :smile: thats not it, just so i can be clear arrow function return the first argument if no curly braces where provided ``` () => "Hello" ``` this function would just returns "Hello" and `mapStateToProps` is just returning an object.

so literary all you have to do is look at this code and copy paste it, im returning an object thats all it is. the same you do for your component when you render you do ` return ( jsx here )` but isntead here you have the arrow function handling the return

```javascript
const mapStateToProps = state => ({
   categories: state.categories
})
```